learn-css-grid-by-building-a-magazine/css/style230.css

```diff
@@ -52,57 +50,16 @@ main {
   row-gap: 1.5rem;
 }
 
-
 .hero {
   grid-column: 1 / -1;
   position: relative;
 }
 
-.hero-img {
+/* Also add `object-fit` property to `cover`. This will told the browser how to resized the image to fit its container.  */
```

Comment:
```
/*
Also add the `object-fit` property and set it to `cover`. 
This will tell the browser how to resize the image to fit its container. 
*/
```